code/script.js

```diff
@@ -1,47 +1,252 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('name-input')
+const sendBtn = document.getElementById('send')
+const form = document.getElementById('name-form')
+const messageSound = new Audio ("assets/message-pop-alert.mp3");
 
 // Global variables, if you need any, declared here
+let questionNumber = 1 //variable for counting which question we are dealing (defined in Figma)
+let flowers = "" //variable for flower to be used in the final message of the bot 
+let address ="" //variable for address to be used in the final message of the bot 
+let cardText ="" //variable for greeting card's greeting to be used in the final message of the bot 
+let delTime = ""//variable for delivery time to be used in the final message of the bot 
+let cardYes = "" //variable for card to be used in the final message of the bot 
 
 // Functions declared here
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
+
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    console.log("user is speaking")
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/smile_user.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
+    console.log("bot is speaking")
```

Comment:
It says in the inspector that line 29 and 39 has an issue, and we think it is because you didn't remove the console.log